docs/_layouts/default.html

```diff
@@ -238,8 +238,8 @@
         {% if page.homepage %}
         <div class="p-strip--image is-dark" style="background-image: url('https://assets.ubuntu.com/v1/775cc62b-vanilla-grad-background.png'); background-position: 75% 50%;">
           <div class="p-content__row">
-            <h1 class="p-heading--stylized">Documentation</h1>
-            <p class="p-heading--four">Everything you need to get started with Vanilla</p>
+            <h1 class="p-heading--stylized">Vanilla documentation</h1>
```

Comment:
p-heading--stylized is no longer needed, as h1 and h2 are already font-weight: 100;

Reply:
@lyubomir-popov Nice! Will update and remove custom `p-heading--stylized` from docs stylesheet 👍